src/fuselage/CCPACSFuselage.h

```diff
@@ -128,6 +128,98 @@ class CCPACSFuselage : public generated::CPACSFuselage, public CTiglRelativelyPo
     TIGL_EXPORT TopoDS_Wire projectConic(TopoDS_Shape wireOrEdge, gp_Pnt origin) const;
     TIGL_EXPORT TopoDS_Wire projectParallel(TopoDS_Shape wireOrEdge, gp_Dir direction) const;
 
+    /*
```

Comment:
I am not very confortable having all these utility function directly in the fuselage class. 

From my point of view they all should be in another class. The only exceptions are methods that require access to private internals that are not accessible using the fuselage's public interface.

Reply:
I understand your point of view in the sense that make the fuselage class heavy. But in the other hand, it will be strange to have, for example, the function "GetSectionCount" (already present before creator age) in fuselage class, but the function "GetNoseCenter" in another place. In this last case, the user need to call these two similar functions from two different places (That seems a bit counter intuitive for me). 
One solution may be to create a child class of CCPACFuselage that can be retrieve from CCPACFuselage. So, the user can at lease ask the two functions "GetSectionCount" and "GetNoseCenter"  from the same object.  This new class can be called CreatorFuselage and contain getter for high level properties and set functions. 
What do you think about this solution?

Reply:
According the discussion of the 8.04.2019 I simplify the class such that CCPACSFuselage class has only the interface that make sense for the general users. 
I create a class "fuselageHelper" that contains the utilities functions needed to perform the  edition operations.
Remark that I remove the function to set a property "Between". 
This kind of edition is allow from the section or element class for one element. So I, will, probably,  create a GUI that allow to select multiple sections at once and edit it.